code/game/machinery/doors/blast_door.dm

```diff
@@ -15,7 +15,7 @@
 	icon = 'icons/obj/doors/rapid_pdoor.dmi'
 	icon_state = null
 	dir = 1
-	closed_layer = ABOVE_WINDOW_LAYER
```

Comment:
There are some subtypes of blast door that will need their layers updated/removed to match this. Check them out one by one

Reply:
So looking at the variants, fortunately they all share the closed_layer variable besides shutter. I also found that Firedoor did some layer weirdness, so gave it the equivalent in define